src/otx/algo/segmentation/litehrnet.py

```diff
@@ -12,7 +12,7 @@
 class LiteHRNet(MMSegCompatibleModel):
     """LiteHRNet Model."""
 
-    def __init__(self, num_classes: int, variant: Literal["18", "s", "x"]) -> None:
+    def __init__(self, num_classes: int, variant: Literal["18", 18, "s", "x"]) -> None:
```

Comment:
It seems 18 is duplicated

Reply:
Modified this to accept `int` as well.

Reply:
As it stands, this doesn't work with `int` 18. But shouldn't 18 be allowed to use `int` from the user's perspective?